package.json

```diff
@@ -39,6 +39,7 @@
     "github": "https://github.com/canjs/canjs"
   },
   "dependencies": {
+    "bit-docs-glob-finder": "0.0.6",
```

Comment:
Actually, this is needed. Otherwise and error is thrown: 
```
/Users/austin/bitovi/canjs/node_modules/bit-docs/lib/configure/node_modules/q/q.js:155
                throw e;
                ^

TypeError: siteConfig.finder is not a function
    at generateOne (/Users/austin/bitovi/canjs/node_modules/bit-docs/lib/generate/generate.js:49:36)
    at generateAndWatch (/Users/austin/bitovi/canjs/node_modules/bit-docs/lib/generate/generate.js:82:10)
    at /Users/austin/bitovi/canjs/node_modules/bit-docs/main.js:6:10
    at _fulfilled (/Users/austin/bitovi/canjs/node_modules/bit-docs/lib/configure/node_modules/q/q.js:854:54)
    at self.promiseDispatch.done (/Users/austin/bitovi/canjs/node_modules/bit-docs/lib/configure/node_modules/q/q.js:883:30)
    at Promise.promise.promiseDispatch (/Users/austin/bitovi/canjs/node_modules/bit-docs/lib/configure/node_modules/q/q.js:816:13)
    at /Users/austin/bitovi/canjs/node_modules/bit-docs/lib/configure/node_modules/q/q.js:624:44
    at runSingle (/Users/austin/bitovi/canjs/node_modules/bit-docs/lib/configure/node_modules/q/q.js:137:13)
    at flush (/Users/austin/bitovi/canjs/node_modules/bit-docs/lib/configure/node_modules/q/q.js:125:13)
    at _combinedTickCallback (internal/process/next_tick.js:67:7)
```